### ISSUE TITLE
Googlemaps is not working with the newest Version.

### ISSUE THREAD

**Issue by Hanfrey** [giansi/gravstrap-theme#13]
Hello,

I just noticed that the google maps plugin is not working with the newest versions. Regarding the googlemaps plugin docs this is because of a missing assets in the new grav core. Maybe you can add it?
Tried it in my environment and worked fine, before was only a grey background.

[Googlemaps Doc](https://github.com/aptly-io/grav-plugin-googlemaps#issues)

```
{% block bottom %}
    {{ assets.js('bottom') }}
{% endblock %}
```

Added the assets.js bottom to the base.html.twig.

**Comment by 4evermaat** [comment]
@Hanfrey, I made an issue [here in the gravstrap github](https://github.com/giansi/gravstrap/issues/4) about a googlemaps bug I noticed.  I think it is the same bug.

Can you pinpoint exactly where these files that you edited are located to get maps to work?

**Comment by giansi** [comment]
There is a bug ìn the latest googlemaps plugin. I'm working on a new shortcode to solve the issue

**Comment by giansi** [comment]
Just released gravstrap plugin v1.0.3 which solves this issue.
